code/modules/gear_presets/_select_equipment.dm

```diff
@@ -831,104 +821,111 @@ var/list/rebel_rifles = list(
 			H.equip_to_slot_or_del(new /obj/item/clothing/head/helmet/marine/veteran/kutjevo(H), WEAR_HEAD)
 
 /datum/equipment_preset/proc/add_random_survivor_equipment(var/mob/living/carbon/human/H) // Think of this gear as something a survivor of an outbreak might get before shortly taking shelter I.E spawn.
-	var/random_gear = rand(0,7)
+	var/random_gear = rand(0,5)
 	switch(random_gear)
 		if(0)
-			H.equip_to_slot_or_del(new /obj/item/explosive/grenade/incendiary/molotov(H), WEAR_R_HAND)
+			H.equip_to_slot_or_del(new /obj/item/explosive/grenade/incendiary/molotov(H), WEAR_IN_BACK)
 		if(1)
-			H.equip_to_slot_or_del(new /obj/item/storage/box/m94(H), WEAR_R_HAND)
+			H.equip_to_slot_or_del(new /obj/item/storage/box/m94(H), WEAR_IN_BACK)
 		if(2)
-			H.equip_to_slot_or_del(new /obj/item/weapon/gun/smg/nailgun(H), WEAR_R_HAND)
+			H.equip_to_slot_or_del(new /obj/item/weapon/gun/smg/nailgun(H), WEAR_IN_BACK)
 		if(3)
-			H.equip_to_slot_or_del(new /obj/item/weapon/melee/twohanded/fireaxe(H), WEAR_R_HAND)
+			H.equip_to_slot_or_del(new /obj/item/weapon/melee/twohanded/fireaxe(H), WEAR_IN_BACK)
 		if(4)
-			H.equip_to_slot_or_del(new /obj/item/tool/weldingtool/largetank(H), WEAR_R_HAND)
+			H.equip_to_slot_or_del(new /obj/item/tool/weldingtool/largetank(H), WEAR_IN_BACK)
 		if(5)
-			H.equip_to_slot_or_del(new /obj/item/storage/firstaid/adv(H.back), WEAR_R_HAND)
+			H.equip_to_slot_or_del(new /obj/item/storage/firstaid/adv(H.back), WEAR_IN_BACK)
 
+/datum/equipment_preset/proc/add_random_prison_survivor_armor(var/mob/living/carbon/human/H) // Random set of armor a prisioner might repurpose in the event of a breakout
```

Comment:
why is the var here called random_weapon if its adding armor, and why is a bayonet one of the options